Source/Urho3D/Core/Context.cpp

```diff
@@ -242,7 +242,7 @@ AttributeHandle Context::RegisterAttribute(StringHash objectType, const Attribut
 {
     // None or pointer types can not be supported
     if (attr.type_ == VAR_NONE || attr.type_ == VAR_VOIDPTR || attr.type_ == VAR_PTR
-        || attr.type_ == VAR_CUSTOM_HEAP || attr.type_ == VAR_CUSTOM_STACK)
+        || attr.type_ == VAR_CUSTOM_HEAP || (attr.type_ == VAR_CUSTOM_STACK && !attr.defaultValue_.IsCustomType<SharedPtr<Serializable>>()))
```

Comment:
This check is getting bulky. Maybe split this if into two?
One to check attr types. And another to check supported custom types.